src/integrationTest/java/uk/gov/hmcts/reform/managecase/api/controller/NoticeOfChangeControllerIT.java

```diff
@@ -55,11 +65,83 @@
 
     private static final String RAW_QUERY = "{\"query\":{\"bool\":{\"filter\":{\"term\":{\"reference\":%s}}}}}";
     private static final String ES_QUERY = String.format(RAW_QUERY, CASE_ID);
+    private static final String QUESTION_ID_1 = "QuestionId1";
+    public static final String ORGANISATION_ID = "QUK822N";
     private final Map<String, JsonNode> caseFields = new HashMap<>();
     private final List<SearchResultViewItem> viewItems = new ArrayList<>();
     private static final String ANSWER_FIELD_APPLICANT = "${applicant.individual.fullname}|${applicant.company.name}|"
-        + "${applicant.soletrader.name}:Applicant";
-
+        + "${applicant.soletrader.name}|${OrganisationPolicy.OrganisationPolicy1"
+        + ".Organisation.OrganisationID}:Applicant";
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @BeforeEach
+    void setUp() throws JsonProcessingException {
+        AuditEvent event = new AuditEvent();
+        event.setEventName("NOC");
+        CaseViewEvent caseViewEvent = CaseViewEvent.createFrom(event);
+        CaseViewResource caseViewResource = new CaseViewResource();
+        CaseViewEvent[] caseViewEventArray = {caseViewEvent};
+        caseViewResource.setCaseViewEvents(caseViewEventArray);
```

Comment:
This needs to be

caseViewResource.setCaseViewActionableEvents

See 

https://github.com/hmcts/aac-manage-case-assignment/pull/120/commits/065d15776b66ba134b706402863f2bdd2c455ad5#diff-f8c851d11caefcafe64db2d16a68a4c2d925e91cb57db1a137e82287ac83342a

where I have fixed both the implementation in NoticeOfChangeService and in corresponding unit and feature tests.

Perhaps we need to discuss how to get this change in